### ISSUE TITLE
Add CA Cert for viewer security encryption

### ISSUE THREAD

**Issue by emperorstarfinder** [Virtual-Universe/Universe-Viewer-Dev#8]
Per an issue found by MOSES grid we should add the CA Cert pieces to enable the encryption for the viewer.  Alchemy does have this as certified by Cinder.

**Comment by CaseyraeStarfinder** [comment]
@emperorstarfinder I agree with you here that the CA certificates need to be added.  This will help allow the viewer to work better with encryption.  There will obviously be other pieces @BriaKallistaStarfinder  and myself will have to change to enable this feature fully.  Bria and I have had this very discussion.  

I received the report you emailed me and have reviewed the comments corresponding to the report from Douglas Maxwell via Hypergrid Business.  I am a little concerned about the method he used and the way he wrote the report.  Any research paper that is legitimate would have the executive summary, the problems to be researched, the findings of the research, and what the solutions to those problems are, and a conclusion.  Mr. Douglas' paper looks more like a publicity stunt even though the security concerns he mentions are legitimate.  I am also a bit surprised he didn't do a scan to see what CVEs and Resource Leaks were in the code as those to cause security vulnerabilities in the code.

This request will be included for the alpha release.

Best Regards,
Caseyrae Starfinder
Governing Body Vice Chair (Europe)
Core Developer
Viewer Developer Work Group Leader
Viewer Development Work Group
Second Galaxy Development Team
Email: caseyrae@secondgalaxy.com
